src/uwtools/drivers/make_hgrid.py

```diff
@@ -0,0 +1,80 @@
+"""
+A driver for chgres_cube.
+"""
+
+from pathlib import Path
+from typing import Optional
+
+from iotaa import asset, dryrun, task, tasks
+
+from uwtools.drivers.driver import Driver
+from uwtools.strings import STR
+
+
+class MakeHgrid(Driver):
+    """
+    A driver for make_hgrid.
+    """
+
+    def __init__(
+        self,
+        config: Optional[Path] = None,
+        dry_run: bool = False,
+        batch: bool = False,
+    ):
+        """
+        The driver.
+
+        :param config: Path to config file (read stdin if missing or None).
+        :param dry_run: Run in dry-run mode?
+        :param batch: Run component via the batch system?
+        """
+        super().__init__(config=config, dry_run=dry_run, batch=batch)
+        if self._dry_run:
+            dryrun()
+
+    # Workflow tasks
+
+    @tasks
+    def provisioned_run_directory(self):
+        """
+        Run directory provisioned with all required content.
+        """
+        yield self._taskname("provisioned run directory")
+        yield self.runscript()
+
+    @task
+    def runscript(self):
+        """
+        The runscript.
+        """
+        path = self._runscript_path
+        yield self._taskname(path.name)
+        yield asset(path, path.is_file)
+        yield None
+        self._write_runscript(path=path, envvars={})
+
+    # Private helper methods
+
+    @property
+    def _driver_name(self) -> str:
+        """
+        Returns the name of this driver.
+        """
+        return STR.makehgrid
+
+    @property
+    def _runcmd(self):
+        """
+        Returns the full command-line component invocation.
+        """
+        executable = self._driver_config["execution"]["executable"]
+        return "%s" % (executable)
```

Comment:
It turns out that we probably don't need this custom implementation: The one inherited from `Driver` should be fine. The one @christinaholtNOAA wrote for `global_equiv_resolv` needed special treatment, but the one @NaureenBharwaniNOAA added for `esg_grid` turned out not to be necessary, and #475 removes it. If you remove this, all your tests should still pass with no other changes -- I'll be very curious to hear if they don't.

Reply:
Ah, thanks, this was from before that was updated - I'll make sure to test through to make sure,